tests/org.codechecker.eclipse.rcp.it.tests/src/org/codechecker/eclipse/plugin/AllTests.java

```diff
@@ -0,0 +1,33 @@
+package org.codechecker.eclipse.plugin;
```

Comment:
This is the new root for the integration tests. The tests will be executed in @SuiteClasses annototatin order, that specified in line 14.